src/lib/account.ts

```diff
@@ -57,24 +69,24 @@ export async function getUserAddress(userSession, username) {
  * Uses the AccountsApi of the stacks blockchain api client library,
  * returns the stacks balance object with property `balance` in decimal.
  */
-export function fetchAccount(addressAsString) {
+export async function fetchAccount(addressAsString: string) {
```

Comment:
Maybe better to specify the function return type in the definition than using `as` on return.

Reply:
that can't happen because the types in the `@stacks/stacks-blockchain-api-types` package are not the same in the `@stacks/blockchain-api-client` package which defines some parts of the response as `object` which isn't useful